tests/data/README.md

```diff
@@ -362,6 +362,22 @@ Box structure of the items in this file:
 [alpha] [alpha]
 ```
 
+## Animated Images
+
+### File [star-animated-8bpc.avif](star-animated-8bpc.avif)
+
+![](star-animated-8bpc.avif)
+
+Source: [Chromium blink tests](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/images/resources/avif/star-animated-8bpc.avif;l=1;drc=c5a2c8c443872070821e51c713845651121f39fb)
```

Comment:
Would it be possible to add the license? I am worried about [this TODO note](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/images/resources/avif/README.md;l=198).

I found [blink/LICENSE_FOR_ABOUT_CREDITS](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/LICENSE_FOR_ABOUT_CREDITS) and [LICENSE](https://source.chromium.org/chromium/chromium/src/+/main:LICENSE) when going up the source tree. Do you know if one of these apply to that file? I guess we can mention both just in case.

Reply:
Good point.

Removed the file from chromium and replaced it with a dummy image generated with ffmpeg.